.gitlab-ci.yml

```diff
@@ -77,6 +77,13 @@ integration-testing:
     - chmod 400 $E2E_PRIVATE_KEY_PATH
     - ssh-add $E2E_PRIVATE_KEY_PATH
     - pip install -r requirements.txt
+    - go install github.com/DataDog/orchestrion@v0.9.4
+    - orchestrion pin
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Could move these to the Dockerfile that defines the test-infra runner image, at the root of this repo